dashboards-observability/public/components/visualizations/charts/financial/gauge/gauge.tsx

```diff
@@ -41,13 +42,8 @@ export const Gauge = ({ visualizations, layout, config }: any) => {
 
   // data config parametrs
   const { dataConfig = {}, layoutConfig = {} } = userConfigs;
-  const dimensions = dataConfig[GROUPBY]
-    ? dataConfig[GROUPBY].filter((item) => item.name !== '')
-    : [];
-  const series = dataConfig[AGGREGATIONS]
-    ? dataConfig[AGGREGATIONS].filter((item) => item.name !== '')
-    : [];
-  const dimensionsLength = dimensions.length;
+  let xaxes = dataConfig[GROUPBY] ? dataConfig[GROUPBY] : [];
```

Comment:
can be written as let xaxes = dataConfig[GROUPBY] ?? [];

Reply:
ok adding the changes